af_backend_tema/__manifest__.py

```diff
@@ -4,24 +4,25 @@
 {
     "name": "AF Backend Tema V12",
     "summary": "AF Backend Theme V12	",
-    "version": "12.0.0.4",
+    "version": "12.0.0.5",
     "category": "Theme/Backend",
     "description": """
 		AF Backend tema för Odoo 12.0 community edition.
 		v12.0.0.3 Added dependancy for the web-module.
+		v12.0.0.5 Reverted Odoo Waffle and favicon icon to normal.
     """,
     "author": "Vertel AB",
     "license": "LGPL-3",
     "installable": True,
     "depends": [
         'base', 'portal', 'web'
     ],
-    'qweb': [
-        'static/src/xml/web.xml',
-    ],
+    # 'qweb': [
+    #     'static/src/xml/web.xml',
+    # ],
     "data": [
         'views/assets.xml',
         'views/template.xml',
-        'views/webclient_template.xml'
+        # 'views/webclient_template.xml'
```

Comment:
Shouldn't this file be removed as well?
This PR just comment out the reference to the file, but the file is still there.